sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/EstimationUtils.scala

```diff
@@ -114,4 +114,197 @@ object EstimationUtils {
     }
   }
 
+  /**
+   * Returns the number of the first bin into which a column values falls for a specified
+   * numeric equi-height histogram.
+   *
+   * @param value a literal value of a column
+   * @param histogram a numeric equi-height histogram
+   * @return the number of the first bin into which a column values falls.
+   */
+
```

Comment:
could you remove the empty line between method comment and its definition?
same for other methods here.